systest/build.rs

```diff
@@ -41,5 +43,6 @@ fn main() {
     cfg.skip_roundtrip(|t| t == "git_clone_options" || t == "git_submodule_update_options");
 
     cfg.skip_type(|t| t == "__enum_ty");
+    cfg.skip_type(|t| t == "git_transfer_progress");
```

Comment:
How come this is skipped?

Reply:
I didn't write this commit. I think @mkeeler wanted to turn off deprecation warning according to the commit message.

Reply:
I think this may still want to get removed?